bluebottle/bluebottle_dashboard/static/jet/css/themes/goodup/_extra.scss

```diff
@@ -91,3 +91,26 @@ span.help + p.help {
     padding-left: 0;
   }
 }
+
+.icon-picker {
+
+}
+
+.icon-picker li{
+  margin: 4px 0;
+}
+
+
+.icon-picker li input{
+  display: none;
+}
+
+.icon-picker li img{
+  padding: 4px;
+  cursor: pointer;
+}
+
+
+.icon-picker li input:checked+img{
+  background-color: #64A03C66;
```

Comment:
Adding to @manuel-goodup feedback, this is no valid colorcode

Reply:
Why not? Because it has opacity?

Reply:
@gannetson RGBA has more browser support than hex-with-opacity: 

https://caniuse.com/?search=rgba
https://caniuse.com/css-rrggbbaa

Reply:
There wasn't any design for this, no colors, an definitely any need for opacity.
I removed it ;-)